chia/seeder/crawler.py

```diff
@@ -60,14 +63,21 @@ def __init__(
         self.best_timestamp_per_peer: Dict[str, int] = {}
         if "crawler_db_path" in config and config["crawler_db_path"] != "":
             path = Path(config["crawler_db_path"])
-            self.db_path = path.resolve()
+            self.db_path = path.expanduser()
         else:
             db_path_replaced: str = "crawler.db"
             self.db_path = path_from_root(root_path, db_path_replaced)
         mkdir(self.db_path.parent)
         self.bootstrap_peers = config["bootstrap_peers"]
         self.minimum_height = config["minimum_height"]
         self.other_peers_port = config["other_peers_port"]
+        self.versions: Dict[str, int] = {}
+        if "minimum_version_count" in self.config and self.config["minimum_version_count"] > 0:
+            self.minimum_version_count = self.config["minimum_version_count"]
+        else:
+            self.minimum_version_count = 100
```

Comment:
This was probably added by me originally - IIRC we didn't even used to have an option for this, but for testnet metrics, we report on versions with a threshold lower than the 100 that used to be hardcoded. Your suggestion here seems reasonable though.. we'll never actually see negative numbers, but if we allow that value, it just means all version counts will be reported, so that should be fine, especially with the warning about the odd configuration value.

Reply:
Maybe `f"Crawler configuration minimum_version_count expected to be greater than zero: {self.minimum_version_count!r}"`.  Sounds nicer than 'abnormal'.  It also expresses what is normal.